tests/Avalonia.RenderTests/Controls/TextBlockTests.cs

```diff
@@ -92,7 +92,7 @@ Control text(VerticalAlignment verticalAlignment, bool clip = true, bool restric
             };
 
             await RenderToFile(target);
-            CompareImages();
+            CompareImages(skipImmediate: true);
```

Comment:
Does this PR somehow breaks immediate mode rendering? It might be worth investigating.

Reply:
It seems that software rendering produces slightly different font rendering not sure why. I will try different edging modes. Ideally  immediate rendering would not use BitmapRenderTarget

Reply:
Render tests are always using software rendering